### DISCUSSION TITLE
Welcome to canal_olympia_scrapping_api Discussions!

### DISCUSSION THREAD

**Discussion by NemesisX1** [NemesisX1/canal_olympia_scrapping_api#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NemesisX1** [comment]
Hello hello @abstractHunter 

I merged your work. I added the tests that were missing. Feel free to get a look a this.

I will also add the current TheaterNameModel in the Swagger docs in order to make it clear in the API docs

One again thank you for your contribution ;-)

**Comment by abstractHunter** [comment]
Hello @NemesisX1, it is a pleasure.
I wanted to update the docs but i didn't figure where to do that 😅😅.

**Reply by NemesisX1** [reply]
Oh don't worry about the docs. It is a swagger / OpenAPI 3.0 documentation powered by the swagger-autogen package.

You will have a complete overview of this way to write docs just here https://swagger-autogen.github.io/docs.

All the current api docs is located in the src/routes folder and in the swagger.ts file at the root of the repository. It can serve you as an example.

Just run "**npm run docs**" to regenerate this current swagger docs with the current changes.

**Reply by abstractHunter** [reply]
OK, thanks for the link.
